### PR TITLE
Extend README with an instruction for the local setup

### LINKED ISSUE #246: Consider extending README with an instruction for the local setup
I think it would be good to have in the README an instruction how to build and validate spec locally. 
Things to consider:
- how to install and run `Bikeshed`
- how to generate and validate CDDL (might be related to #223)

### PR DESCRIPTION
**Pull request by lutien**
Closes #246

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,30 @@ WebDriver BiDi is not ready. Here's what we have so far:
...
+```
+./scripts/build.sh --install
+```
+This script will install `bikeshed` and generate an `index.html` file for the specification.
+
+Later on, you can omit `--install` argument to skip the installation step.
+
+## How to generate CDDL locally
+
+Make sure you have `npm` and `rust` installed.
```
Comment (whimboo) [inline README.md:28]: It might be good to link to the appropriate install documentation here as well (similar to what you did for `bikeshed`).

**Review by whimboo — CHANGES_REQUESTED** [review]
Thank you @lutien! IMHO it's a great help in how to get started contributing to the BiDi spec. Also given that the README is small enough the used location should be fine.

There is just a minor nit from my side.

File: README.md (hunk 1 above)
Comment (lutien) [reply README.md]: sure, added links to npm and rust installation instructions

**Review by whimboo — APPROVED** [review]
Looks fine to me. Thanks!

@jgraham mind doing a final review in case something is missing?

**Pull request merged**